File: Pattern13.java
import java.util.Scanner;

public class Pattern13 {
    public static void main(String args[]) {

        System.out.println("enter number:");
        Scanner inp = new Scanner(System.in);
        int n = inp.nextInt();

        for (int i = 1; i <= n; i++) {

            for (int k = n-i; k >= 1; k--) {

                System.out.print(" ");
            }

            for (int j = 1; j <= i; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: PATTERNS/Pattern13/Pattern13.go
package main

import "fmt"

func main(){
	var n int
	fmt.Print("Enter the row number:")
	fmt.Scanf("%d", &n)

	for i:=1;i<=n;i++{
		for k:=i;k<=n;k++{
			fmt.Print(" ")
		}
		for j:=1;j<=i;j++{
			fmt.Print("*")
		}
		fmt.Println()
		
	}
}